tools/flang2/flang2exe/llutil.cpp

```diff
@@ -2194,6 +2194,79 @@ metadata_args_need_struct(void)
   return ll_feature_metadata_args_struct(&llvm_get_current_module()->ir);
 }
 
+/**
+ * This function returns true for the types supported
+ * in function make_param_op
+ */
+bool should_preserve_param(SPTR sptr) {
```

Comment:
can this be marked as a constant function?

Reply:
Sure, I shall make that.